frontend/src/components/ProjectList/index.tsx

```diff
@@ -12,6 +13,10 @@ const ProjectsList: FC<Props> = ({ projects }) => {
 
   return (
     <>
+      <h1>Datasets</h1>
+      <p>
+        Explore public dataset or add your own. <Link to="">Learn more</Link>
```

Comment:
Ah sorry Trent, can we use the following instead please 😄 🙏 ?

>The cellxgene data portal is a repository of public, explorable single-cell datasets. If you have a public dataset which you would like hosted for visualization on this site, please let us know at cellxgene@chanzuckerberg.com.

I think that's what Ambrose wants in the ticket. Thank you!

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/corpora-data-portal/607

Reply:
Fixed <img width="1091" alt="Screen Shot 2020-09-16 at 10 10 02 AM" src="https://user-images.githubusercontent.com/1429913/93369851-d1b3ca80-f804-11ea-9a4b-b1a57a112b60.png">